src/storage/raw.c

```diff
@@ -116,16 +134,50 @@ raw_init()
                                   sizeof("out.raw"),
                                   0,
                                   0,
-                                  pixel_scale_um,
-                                  0));
-    self->writer = (struct Storage){ .state = DeviceState_AwaitingConfiguration,
-                                     .set = set,
-                                     .get = get,
-                                     .start = start,
-                                     .append = append,
-                                     .stop = stop,
-                                     .destroy = destroy };
+                                  pixel_scale_um));
+    self->writer =
+      (struct Storage){ .state = DeviceState_AwaitingConfiguration,
+                        .set = raw_set,
+                        .get = raw_get,
+                        .get_meta = raw_get_meta,
+                        .start = raw_start,
+                        .append = raw_append,
+                        .stop = raw_stop,
+                        .destroy = raw_destroy,
+                        .reserve_image_shape = raw_reserve_image_shape };
     return &self->writer;
 Error:
     return 0;
 }
+
+#ifndef NO_UNIT_TESTS
+
+#ifdef _WIN32
+#define acquire_export __declspec(dllexport)
+#else
+#define acquire_export
+#endif
+
+acquire_export int
+unit_test__raw_get_meta()
```

Comment:
Consider making this a standalone test in the tests directory instead of a unit test. 

See how list-devices is done in this repo for an example. It doesn't rely on the use of runtime. Instead it uses the hal to check things.